test/clj/rems/api/test_administration.clj

```diff
@@ -1,90 +1,88 @@
 (ns ^:integration rems.api.test-administration
   "Tests for invariants across administration APIs."
   (:require [clojure.test :refer :all]
+            [rems.api.testing :refer :all]
             [rems.db.core :as db]
-            [rems.api.testing :refer :all]))
+            [rems.db.testing :refer [owners-fixture +test-api-key+]]))
 
 (use-fixtures
   :once
-  api-fixture)
+  api-fixture-without-data
+  owners-fixture)
 
-(def api-key "42")
 (def user-id "owner")
```

Comment:
Could we just inline this? I think it's merely confusing us as to who does things (user vs. owner).

Reply:
sure